modules/multilanguage/src/main/resources/logback.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
This works for now, but might be worth a follow up to allow the end user to configure log levels, log format, etc... for the grpc server as well.